source/css/components/_featured_link.scss

```diff
@@ -37,4 +37,21 @@
       margin-top: $gutter;
     }
   }
+
+  &__learn-more {
+    @extend .btn;
+    @extend .btn--solid;
+    @extend .btn--pro;
+
+    font-weight: 600;
+
+    color: #fff;
+    border-color: #F5A623;
```

Comment:
I think we have colors for this?  `$colour__primary` or something.

Reply:
Will clean this up